File: src/main/java/com/cocktailsguru/app/cocktail/domain/ingredient/IngredientType.java
package com.cocktailsguru.app.cocktail.domain.ingredient;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity(name = "coctail_ingred_category")
@Data
public class IngredientType {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private String name;
}

File: src/main/java/com/cocktailsguru/app/cocktail/domain/ingredient/IngredientType.kt
package com.cocktailsguru.app.cocktail.domain.ingredient

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

@Entity(name = "coctail_ingred_category")
data class IngredientType(
        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        val id: Int,
        var name: String
)
